maestro-orchestra/src/test/java/maestro/orchestra/MaestroCommandSerializationTest.kt

```diff
@@ -78,7 +79,8 @@ internal class MaestroCommandSerializationTest {
                 "retryIfNoChange" : false,
                 "waitUntilVisible" : true,
                 "longPress" : false,
-                "label" : "My TapOnPoint"
+                "label" : "My TapOnPoint",
+                "optional" : false
```

Comment:
please use trailing commas (they haven't been used in this repo before but we're trying to migrate to using them everywhere)

(this applies to all code in this PR)

Reply:
This is JSON which does not support trailing commas I think 🤔 
I used trailing commas in all Kotlin code. Is that what you meant?

Reply:
Yes, I meant Kotlin code. Sorry for the mistake.